### DISCUSSION TITLE
Using alpine

### DISCUSSION THREAD

**Discussion by chrislovecnm** [google/s2geometry#381] [Q&A]
So we are able to get the pip package installed on Debian, but python slim has open CVE's that won't get addressed.

So the other option is alpine.

I found https://pkgs.alpinelinux.org/package/edge/community/x86/s2geometry

But that build is patching OpenSSL https://git.alpinelinux.org/aports/tree/community/s2geometry/fix-newer-openssl.patch

Needless to say running a patch to OpenSSL in production is a hard sell to any security group.

Any options?

**Comment by jmr** [comment]
> But that build is patching OpenSSL https://git.alpinelinux.org/aports/tree/community/s2geometry/fix-newer-openssl.patch
> 
> Needless to say running a patch to OpenSSL in production is a hard sell to any security group.

The name is misleading, but this is a patch to S2, not OpenSSL.

**Reply by chrislovecnm** [reply]
You are correct, my bad.
